src/NHibernate/Driver/ReflectionBasedDriver.cs

```diff
@@ -46,12 +46,12 @@ protected ReflectionBasedDriver(string providerInvariantName, string driverAssem
 
 			if (connectionType == null || commandType == null)
 			{
-#if NETFX || NETSTANDARD2_1_OR_GREATER
+#if NETFX || NETSTANDARD2_1_OR_GREATER || NET5_0_OR_GREATER 
```

Comment:
Then as a consequence of the comment on below lines :

```suggestion
#if NETFX || NETSTANDARD2_1_OR_GREATER || NETCOREAPP2_1_OR_GREATER
```

or 

```suggestion
#if NETFX || NETSTANDARD2_1_OR_GREATER || NET6_0_OR_GREATER
```